public/app/features/gops/configuration-tracker/irmHooks.ts

```diff
@@ -125,175 +126,242 @@ export function useGetEssentialsConfiguration(): EssentialsConfigurationData {
     locationService.push(urlToGoWithIntegration);
   }
 
-  const essentialContent: SectionsDto = {
-    sections: [
+  function getGrafanaAlertingConfigSteps(): SectionDtoStep[] {
+    let steps: SectionDtoStep[] = [
       {
-        title: 'Detect',
-        description: 'Configure Grafana Alerting',
-        steps: [
-          {
-            title: 'Update default contact point',
-            description: 'Update the default contact point to a method other than the example email address.',
-            button: {
-              type: 'openLink',
-              urlLink: {
-                url: `/alerting/notifications`,
-                queryParams: { search: defaultContactpoint, alertmanager: 'grafana' },
-              },
-              label: 'Edit',
-              labelOnDone: 'View',
-              urlLinkOnDone: {
-                url: `/alerting/notifications`,
-              },
-            },
-            done: isContactPointReady(defaultContactpoint, contactPoints),
+        title: 'Update default e-mail contact point',
```

Comment:
I had previously changed this text to not mention email - as although it defaults to having email as the contact point, it doesn't have to be once they've changed it. Could we change it back please?

Reply:
OK, I'll revert it then. @raphael-batte this was in your mockup, but we will be discard this e-mail mention

Reply:
Reverted

Reply:
Thank you! 🙏